### ISSUE TITLE
Add crazy egg code snippet to vanillaframework.io

### ISSUE THREAD

**Issue by kwm14** [canonical-websites/vanillaframework.io#90]
## Summary

Add crazy egg code snippet to vanillaframework.io website, so we can perform A/B testing of new sections and will provide insight on how the site is performing.

## Process

Please add the following tracking script to all pages:
```
<script type="text/javascript" src="//script.crazyegg.com/pages/scripts/0011/8875.js" async="async"></script>
```

- https://vanillaframework.io/
- https://vanillaframework.io/accessibility
- https://vanillaframework.io/browser-support
- https://vanillaframework.io/coding-standards
- https://vanillaframework.io/contribute

**Comment by kwm14** [comment]
Great! Thanks @nel-co

**Comment by anthonydillon** [comment]
This landed as part of #92
